.evergreen/run-tests.sh

```diff
@@ -161,15 +161,15 @@ $PYTHON -c 'import sys; print(sys.version)'
 # files in the xunit-results/ directory.
 
 # Run the tests with coverage if requested and coverage is installed.
-# Only cover CPython. Jython and PyPy report suspiciously low coverage.
+# Only cover CPython. PyPy reports suspiciously low coverage.
+PYTHON_IMPL=$($PYTHON -c "import platform; print(platform.python_implementation())")
 COVERAGE_OR_PYTHON="$PYTHON"
 COVERAGE_ARGS=""
 if [ -n "$COVERAGE" -a $PYTHON_IMPL = "CPython" ]; then
     COVERAGE_BIN="$(dirname "$PYTHON")/coverage"
```

Comment:
This is no longer used?

Reply:
Correct. Updated to remove the now unused vars.